src/components/sbb-radio-button-group/sbb-radio-button-group.tsx

```diff
@@ -0,0 +1,182 @@
+ */
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-radio-button-group.scss',
+  tag: 'sbb-radio-button-group',
+})
+export class SbbRadioButtonGroup {
+  /**
+   * Id of the radio group element.
+   */
+  @Prop() public sbbRadioButtonGroupId = `sbb-radio-button-group-${++nextId}`;
+
+  /**
+   * Id of the radio group element - default name will be auto-generated.
+   */
+  @Prop() public name?: string = `${this.sbbRadioButtonGroupId}-name`;
+
+  /**
+   * Whether the radios can be deselected.
+   */
+  @Prop() public allowEmptySelection = false;
+
+  /**
+   * Whether the radio group is disabled.
+   */
+  @Prop({ reflect: true }) public disabled = false;
+
+  /**
+   * Whether the radio group is required.
+   */
+  @Prop({ reflect: true }) public required = false;
+
+  /**
+   * The value of the radio group.
+   */
+  @Prop({ mutable: true, reflect: true }) public value?: any | null;
+
+  /**
+   * Size variant, either m or s.
+   */
+  @Prop({ reflect: true }) public size?: InterfaceSbbRadioButtonGroup['size'] = 'm';
+
+  /**
+   * State of listed named slots, by indicating whether any element for a named slot is defined.
+   */
+  @State() private _namedSlots = createNamedSlotState('error');
+
+  @Element() private _element!: HTMLElement;
+
+  @Watch('value')
+  public valueChanged(value: any | undefined): void {
+    this._updateRadios();
+    this.didChange.emit({ value });
+  }
+
+  /**
+   * Emits whenever the radio group value changes.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'sbb-radio-button-group_did-change',
+  })
+  public didChange: EventEmitter<any>;
+
+  public connectedCallback(): void {
+    this._updateRadios();
+    this._namedSlots = queryAndObserveNamedSlotState(this._element, this._namedSlots);
+  }
+
+  @Listen('sbbNamedSlotChange', { passive: true })
+  public handleSlotNameChange(event: CustomEvent<Set<string>>): void {
+    this._namedSlots = queryNamedSlotState(this._element, this._namedSlots, event.detail);
+  }
+
+  @Listen('sbb-radio-button_did-select', { passive: true })
+  public onRadioButtonSelect(event: CustomEvent<Set<string>>): void {
+    this.value = event.detail;
+  }
+
+  private _updateRadios(): void {
+    const radios = this._radioButtons;
+    const value = this.value ?? radios.find((radio) => radio.checked)?.value;
+
+    for (const radio of radios) {
+      radio.name = this.name;
+      radio.checked = radio.value === value;
+      radio.disabled = radio.disabled ? radio.disabled : this.disabled;
+      radio.required = radio.required ? radio.required : this.required;
+      radio.tabIndex = radio.checked && !radio.disabled ? 0 : -1;
+      radio.size = this.size;
+      radio.allowEmptySelection = this.allowEmptySelection;
+    }
+
+    // If no option is selected, make the first one focusable
+    radios.length && (radios[0].tabIndex = value || radios[0].disabled ? radios[0].tabIndex : 0);
+  }
+
+  private get _radioButtons(): InterfaceSbbRadioButton[] {
```

Comment:
until now we never used getters. Not sure if we should start with it or not :P

Reply:
We have one place where we use it in the tabs: `private get _enabledTabs()`. I remember I was suggested to use it there. For me it's ok also not to use it, you tell me.